CloudWorkshop_DIPC_200.md

```diff
@@ -177,35 +173,27 @@ Up until this point, we have monitored the job within DIPC but it would nice to
     SELECT COUNT(*) REGION FROM SRC_REGION;
     SELECT COUNT(*) SALES_PERSON FROM SRC_SALES_PERSON;
     ```
-![](images/200/image200_35.png)
-3.	Execute the statements by clicking on the “Run script” icon (second icon from left to right on the icon bar; right-ponting green arrow head on top of a page)
-4.	This will show all entities count on the results panel (lower section) 
-![](images/200/image200_36.png)
-5.	Repeat steps 1 through 4 for connection “WS - SALES_TRG ”     
+![](images/200/image200_35p.png)
+3.	Execute the statements by clicking on the “Run script” icon (second icon from left to right on the icon bar; right-ponting green arrow head on top of a page).
+4.	This will show all entities count on the results panel (lower section). 
+![](images/200/image200_36p.png)
+5.	Repeat steps 1 through 4 for target database (for example "WS - SALES_TGT").     
```

Comment:
Actually, what we tell them to create (in the Appendix) is SALES_TRG. If the image is incorrect, would you get a correct one please?